Extensions/Leaderboards/leaderboardstools.ts

```diff
@@ -233,30 +227,25 @@ namespace gdjs {
                 response.statusText
               );
               scoreSavingState.setError(errorCode);
-              errorVar.setString(errorCode);
               return;
             }
 
             scoreSavingState.closeSaving();
 
             return response.text().then(
```

Comment:
This can be removed I think

Reply:
There is condition to check a failure. An expression can be added later to give more details about the error, but this is not mandatory.

Reply:
That's ok with me

Reply:
Yes I think it's good to keep 👍